name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages_de.properties

```diff
@@ -525,9 +525,9 @@ LabelLevelNameNumber = {0} (Ebene {1})
 
 LabelLevelNumber = Ebene {0}
 
-LabelMaxDrawdown = Maximum Drawdown
+LabelMaxDrawdown = Maximaler Drawdown
 
-LabelMaxDrawdownDuration = Maximum Drawdown Duration
+LabelMaxDrawdownDuration = L\u00E4nge des maximalen Drawdown
```

Comment:
BTW, die "Maximum Drawdown Duration" muss nicht die Zeitspanne gewesen sein, in der der Maximum Drawdown stattgefunden hat. Es ist halt die längste Zeitspanne, die es braucht um wieder auf ein Peak zu kommen.

> Many assume Max DD Duration is the length of time between new highs during which the Max DD (magnitude) occurred. But that isn’t always the case. The Max DD duration is the longest time between peaks, period. So it could be the time when the program also had its biggest peak to valley loss (and usually is, because the program needs a long time to recover from the largest loss), but it doesn’t have to be.
> Quelle: http://en.wikipedia.org/wiki/Drawdown_(economics)

Deswegen finde ich "Länge des maximalen Drawdown" etwas irreführend. Aber einen deutschen Begriff habe ich leider nicht gefunden.


Den Begriff "Maximum Drawdown" haben ich wegen dem Deutschen Wikipedia Eintrag genommen: http://de.wikipedia.org/wiki/Maximum_Drawdown Und für "maximaler Drawdown" finde ich auch viel weniger Google Suchergebnisse. Hast Du eine gute Quelle? Mir fehlen da sowieso gute (deutsche) Links.